### ISSUE TITLE
Understanding auto-sklearn results

### ISSUE THREAD

**Issue by ran1937** [automl/auto-sklearn#268]
Hi,

I have been running auto-sklearn with one instance for one hour and 6 instances in parallel for one hour, but the accuracy obtained stayed the same. Thus, I was wondering whether there are any built-in tools/hooks which could give me more information besides just the accuracy obtained:
- maybe see a percentage of all pre-processing steps/algorithms that were tried - should I let it run for longer so that more of them could be tried?
- maybe any other information used in algorithm selection that could help determine if running any longer could produce better results?

Thank you.

**Comment by mfeurer** [comment]
Hi,

For the second question you could call [show_models()](http://automl.github.io/auto-sklearn/stable/api.html#autosklearn.classification.AutoSklearnClassifier.show_models) which gives prints the final ensemble. Also, you can have a look at the attribute `cv_results_` of the [AutoSklearnClassifier](http://automl.github.io/auto-sklearn/stable/api.html#autosklearn.classification.AutoSklearnClassifier) which shows all runs tried so far (in this instance of Auto-sklearn). There is no function for your first bullet point.

Best,
Matthias

**Comment by Motorrat** [comment]
You can check [in autosklearn-zeroconf](https://github.com/paypal/autosklearn-zeroconf/blob/master/zeroconf.py#L126) code how I determine the runtime based on a heuristic. It has shown good results for me on different datasets so far. If you will go for implementation of the cv_results_ based limit - let me know. I'd be interested in a collaboration.

**Comment by mfeurer** [comment]
@ran1937 do you have further questions or can this be closed for now?

**Comment by kaiweiang** [comment]
@Motorrat Hi, I can't access the link, https://github.com/paypal/autosklearn-zeroconf/blob/master/zeroconf.py#L126
Could you provide the working link? 

Thanks a lot

**Comment by Motorrat** [comment]
@kaiweiang  https://github.com/paypal/autosklearn-zeroconf/blob/da586ab6a5c6a571ac4f8b9c433a0e7c63b57129/README.md#how-does-it-work and ultimately https://github.com/paypal/autosklearn-zeroconf/blob/b6b7c2d2a3531c645c6d739558f885382273b7d9/bin/dataTransformationProcessing.py#L37 
Feel free to ask for more explanation after you have read the README and consulted with the code.

**Comment by kaiweiang** [comment]
@Motorrat Thanks for the reply! I'll take a look at them later.
